bindata/manifests/sriov-config-service/kubernetes/sriov-config-service.yaml

```diff
@@ -1,14 +1,15 @@
 contents: |
   [Unit]
-  Description=Configures SRIOV NIC
-  After=network-pre.target
-  Before=NetworkManager.service kubelet.service
-  
+  Description=Configures SRIOV NIC - pre network configuration
+  DefaultDependencies=no
+  After=network-pre.target systemd-udev-settle.service systemd-sysusers.service systemd-sysctl.service
+  Before=network.target NetworkManager.service systemd-networkd.service ovs-vswitchd.service ovsdb-server.service
```

Comment:
Note: If a service defined in After or Before section is not exist it will be simply ignored.